codegen/java-gen/src/test/scala/com/lightbend/akkasls/codegen/java/ActionTestKitGeneratorSuite.scala

```diff
@@ -0,0 +1,165 @@
+/*
+ * Copyright 2021 Lightbend Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.lightbend.akkasls.codegen
+package java
+
+import com.google.protobuf.DescriptorProtos.FileDescriptorSet
+import com.google.protobuf.DescriptorProtos
+import com.google.protobuf.Descriptors
+
+import _root_.java.io.FileInputStream
+import _root_.java.nio.file.Paths
+import scala.jdk.CollectionConverters._
+import scala.util.Using
+import com.google.protobuf.ExtensionRegistry
+import org.slf4j.LoggerFactory
+
+import scala.collection.mutable
+import scala.util.Try
+
+class ActionTestKitGeneratorSuite extends munit.FunSuite {
+  private val testData = TestData.javaStyle
+
+  test(
+    "it can generate an specific TestKit for the proto files " +
+    "in test/resources/testkit") {
+
+    val packageName = "com.example.actions"
+    val model = generateModel
+    val className = "CounterJournalToTopicAction"
+    val service: ModelBuilder.ActionService = {
+      model.get.services("com.example.actions.CounterJournalToTopic").asInstanceOf[ModelBuilder.ActionService]
+
+    }
+    val entity: ModelBuilder.EventSourcedEntity = {
+      model.get.entities("com.example.domain.Counter").asInstanceOf[ModelBuilder.EventSourcedEntity]
+    }
+
+    //This prints the contents of the model, services or entity. Necessary to
```

Comment:
This is a new approach to create the initial `BuilderModel.Model` by reading the `user-function.desc` file from one of the samples. Then we can inspect its content by printing it with `munit.Assertions.munitPrint(model)`.